python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -618,6 +733,7 @@ def build_sensor_context(
     repository_name: Optional[str] = None,
     repository_def: Optional["RepositoryDefinition"] = None,
     sensor_name: Optional[str] = None,
+    resource_defs: Optional[Mapping[str, "ResourceDefinition"]] = None,
```

Comment:
Recommend naming this `resources` as opposed to `resource_defs` to match with `build_op_context`, `build_resource_context`, etc